rero_ils/modules/documents/views.py

```diff
@@ -598,10 +597,13 @@ def online_holdings(document_pid, viewcode='global'):
         .filter('term', _masked=False)
     if organisation:
         query = query.filter('term', organisation__pid=organisation.pid)
-    results = query.source(['library', 'electronic_location', 'notes']).scan()
-    holdings = []
+    results = query.source(['library', 'electronic_location',
+                            'enumerationAndChronology', 'notes']).scan()
+
+    holdings = {}
     for record in results:
         library = Library.get_record_by_pid(record.library.pid)
+        library_holdings = holdings.get(library['name'], [])
```

Comment:
Don't think that default value should be an array ? `library['name']` return a string isn't it ?

Reply:
I need to retrieve an array of holdings by library name or an empty array if there are no holdings